packages/render-fidelity-tools/src/components/renderers/dspbr-pt-viewer.ts

```diff
@@ -80,29 +80,29 @@ export class PathtracingViewer extends LitElement {
     this[$renderer] = new PathtracingRenderer(this[$canvas]!, enableControls);
 
     const renderer = this[$renderer];
-    renderer.autoScaleOnImport(false);
     renderer.setPixelRatio(0.5);
```

Comment:
The value of 0.5 forces the renderer to cut the rendering resolution in half. You may want to bump this up to 1.0, but  this will significantly increase render time and gpu memory consumption.